src/app/pages/OpenWalletPage/webextension.tsx

```diff
@@ -2,9 +2,16 @@ import React from 'react'
 import { useHref } from 'react-router-dom'
 import { openLedgerAccessPopup } from 'utils/webextension'
 import { SelectOpenMethod } from './'
+import { FromLedger } from './Features/FromLedger'
 
 export function OpenWalletPageWebExtension() {
   const href = useHref('connect-device')
 
   return <SelectOpenMethod webExtensionLedgerAccess={() => openLedgerAccessPopup(href)} />
 }
+
+export function FromLedgerWebExtension() {
+  const href = useHref('/open-wallet/connect-device')
+
+  return <FromLedger webExtensionLedgerAccess={() => openLedgerAccessPopup(href)} disableBluetoothLedger />
```

Comment:
not sure we need this `FromLedgerWebExtension` wrapper and `disableBluetoothLedger` prop in `FromLedger`
it can be replaced by runtime check in a component or whatever function is responsible for validating access to ble 
```
import { runtimeIs } from 'config'
const isExtension = runtimeIs === 'extension'
```